code/js/rcalc-loader.js

```diff
@@ -6,6 +6,8 @@ function loadJsScript(jsFilePath) {
   document.getElementsByTagName("head")[0].appendChild(jsScript);
```

Comment:
Currently you are using jquerry. You don't need call document.getElementById etc. ... look here and change your code. https://learn.jquery.com/using-jquery-core/faq/how-do-i-select-an-item-using-class-or-id/

Reply:
yes I fully agree... but we have loader long before localization and before I started using jquery... first time I am using jquery is in localization.js... so I would suggest to leave old code is it is and from now on using localization... how about that?

Reply:
I know ... so you change it if now you are using needed libraries. Write less, do more. 
Remember one thing ... if sometimes find something what is wrong it is not related with current job. FIX IT or CREATE NEW TASK FOR IT. Always be better ... sentence from one Romanian expert on Devoxxx conference last year in Krakow.

Reply:
well well well very wise words from Romania... just one suggestion from my side... my loader loads jquery... meaning... jquery wont work until is loaded, so I can not use jquery in loader

Reply:
Of course in this case it make sense use it by standard way.
He said also that if you are junior person in the team and you have better idea how to solve something which was done by perfect senior ... don't worry and DO IT! ... so just advice for life ;-)

Reply:
thank you, this is very important advice to me... will remeber